sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1574,11 +1570,10 @@ synchronized void enqueueDownloadOrShowUnknownSourcesDialog(final ReleaseDetails
         if (releaseDetails == mReleaseDetails) {
             if (InstallerUtils.isUnknownSourcesEnabled(mContext)) {
                 AppCenterLog.debug(LOG_TAG, "Schedule download...");
-                if (releaseDetails.isMandatoryUpdate()) {
-                    showDownloadProgress();
-                }
-                mCheckedDownload = true;
-                mDownloadTask = AsyncTaskUtils.execute(LOG_TAG, new DownloadTask(mContext, releaseDetails));
+                resumeDownload();
```

Comment:
Because the PR is so big, it's hard to understand this block and follow everything, can you explain?